pandas/tests/extension/test_arrow.py

```diff
@@ -1619,6 +1619,12 @@ def test_add_series_with_extension_array(self, data, request):
                     reason=f"add_checked not implemented for {pa_dtype}",
                 )
             )
+        elif pa.types.is_signed_integer(pa_dtype):
```

Comment:
I think the xfail needs to only be for int8 as the test is failing due to an overflow

Reply:
Fixed